LinaSaysHello/Program.cs

```diff
@@ -13,15 +13,72 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            //Console.WriteLine("Hi! What’s your name?");
-            //string name = Console.ReadLine();
-            //Console.WriteLine("Nice to meet you " + name);
-            //Console.WriteLine("Anastasiya");
-            //Console.WriteLine("Vention");
-            //Console.ReadLine();
+            //Task 1//
+            Console.Write("Enter first number: ");
+            int enteredNr1 = Convert.ToInt32(Console.ReadLine());
 
-           
+            Console.Write("Enter second number: ");
+            int enteredNr2 = Convert.ToInt32(Console.ReadLine());
 
+            Console.Write("Enter one aritmetic operation: +, -, *, /");
+            char aritmeticOperation = Convert.ToChar(Console.ReadLine());
+
+            switch (aritmeticOperation)
+            {
+                case '+':
+                    {
+                        Console.WriteLine("{ 0} + { 1} = {2}", enteredNr1, enteredNr2, aritmeticOperation);
+                    }
+                    break;
+
+                case '-':
+                    {
+                        Console.WriteLine("{ 0} - { 1} = {2}", enteredNr1, enteredNr2, aritmeticOperation);
+                    }
+                    break;
+                case '*':
+                    {
+                        Console.WriteLine("{ 0} * { 1} = {2}", enteredNr1, enteredNr2, aritmeticOperation);
+                    }
+                    break;
+                case '/':
+                    {
+                        Console.WriteLine("{ 0} / { 1} = {2}", enteredNr1, enteredNr2, aritmeticOperation);
+                    }
+                    break;
+                default :
+                    Console.WriteLine("Operation is not present in the list of allowed operations");
+                    break;
+            }
+        
+
+            //Task 2//
+            Console.Write("Enter a random number: ");
+            int a = Convert.ToInt32(Console.ReadLine());
+            if (a == 0 || a == 15)
+            {
+                Console.WriteLine("Partly working!");
+            }
+            else if (a != 37)
+            {
+                Console.WriteLine("Working!");
+            }
+            else if (a < 50 || a >= 32)
```

Comment:
Why did you write the conditions for the "Working!" case in different else..if?

Reply:
In addition, the number entered by the user must satisfy three conditions at once, not just one of them. So you should think about another operator instead of || :)

![image](https://github.com/Linases/LinaSaysHello/assets/55654947/188fc146-f2df-4d50-be44-85450b56a31d)